training/nvidia-bootc/duplicated/ilab-wrapper/ilab

```diff
@@ -60,23 +60,30 @@ fi
 # In the future, we will run podman as the current user, once we figure a
 # reasonable way for the current user to access the root's user container
 # storage.
-CURRENT_USER_NAME=$(id --user --name)
-CURRENT_USER_SUBUID_RANGE=$(awk \
-    --field-separator ':' \
-    --assign current_user="$CURRENT_USER_NAME" \
-    --assign current_uid="$UID" \
-    '$1 == current_user || $1 == current_uid {print $2 ":" $3}' \
-    /etc/subuid)
+if [[ "$UID" == 0 ]]; then
```

Comment:
This should not be part of this PR.

Reply:
I disagree, it's syncing the duplicated wrapper as expected from the hook

Reply:
But looks like #731 already did that